torchvision/datasets/imagenet.py

```diff
@@ -118,41 +114,97 @@ def extra_repr(self):
         return "Split: {split}".format(**self.__dict__)
 
 
-def parse_devkit(root):
-    idx_to_wnid, wnid_to_classes = parse_meta(root)
-    val_idcs = parse_val_groundtruth(root)
+def parse_devkit_archive(archive, meta_file=None):
```

Comment:
Should we change the three `parse_*_archive` to static methods of the dataset? Could have some value since they are now part of the official usage pipeline and the user doesn't have to import them separately.